client/src/components/AuthForm/SignUpForm/SignUpForm.js

```diff
@@ -67,21 +70,48 @@ const SignUpForm = ({
             onChange={handleChange}
             withButton={false}
             icon={<ErrorIcon />}
+            classInput={
+              signUpForm.email !== '' ? styles.activeInput : styles.defaultInput
```

Comment:
this can be converted to: 

signUpForm.email ? styles.activeInput : styles.defaultInput